### ISSUE TITLE
Write a blog post on testing for CRLF Injecting using Akto.

### ISSUE THREAD

**Issue by Ankita28g** [akto-api-security/akto#638]
Akto is an open source API security product.

Your task is to write a blog post on testing for CRLF Injecting using Akto.

### 🎯 Requirements

- Your article has to be publicly available.
- Your article must tag Akto in any way (hashtag, embedded, link...).
- Your article should be at least 1000 words long.
- Your article should look nice. 👀 Use titles, subtitles, screenshots, images, gifs, or even memes.

### ✅ Task summary:

- Drop a comment on this issue indicating that you’re working on it.
- Write a blog post with title `'How I tested for CRLF Injection with Akto'`
- Publish an article on your favorite platform or website. (Medium, [Dev.to](http://dev.to/), Hashnode...)
- Submit a pull request here.
- Share your work on social media and tag https://github.com/akto-api-security/akto (Reddit / HackerNews / Twitter / Facebook / Linkedin)

### 🙋🏼‍♂️ Questions:

If you have questions, need any help, or just want to hang out, make sure to join us on our [[Discord server](https://discord.com/invite/Wpc6xVME4s)](https://discord.com/invite/Wpc6xVME4s).

##

**Comment by falconcode16** [comment]
Looks interesting. You can assign this to me. 
Also, you can have a look at my work :) [(https://medium.com/@rohitpatil1625)]

**Comment by falconcode16** [comment]
Looks interesting. You can assign it to me.
Also, you can have a look at my work :)[Medium](https://medium.com/@rohitpatil1625)

**Comment by Ankita28g** [comment]
Assigned to you @falconcode16 

There are more issues for blogs and tutorials if you want to write. The more PRs you submit, the more opportunity you have to win swags. 😀

**Comment by Ankita28g** [comment]
This issue can have multiple assignees.

**Comment by aayushii9602** [comment]
Can I contribute too? you can check my previous work on,[medium](https://medium.com/@aayushiiiii09)

**Comment by Ankita28g** [comment]
Absolutely! GO ahead and Contribute. Just make sure whatever you write is accurate :)

Let me know if you need my help. Happy Hacktoberfest 🎉

**Comment by aayushii9602** [comment]
I've completed the blog, you can check the Medium link](https://medium.com/@aayushiiiii09/how-i-tested-for-crlf-injection-with-akto-3ba0c832871a)

**Comment by falconcode16** [comment]
Hey @Ankita28g have a look at the article that I have written [Medium](https://medium.com/@rohitpatil1625/how-i-tested-for-crlf-injection-with-akto-2bd24bda047f)

**Comment by Ankita28g** [comment]
Hey @aayushii9602 

Thanks for your submission. I read the blog. Most of the content is incorrect. I can accept the PR once you solve these issues:

1. `akto test my_crlf_injection_test.rb` there is no such command to run test in Akto. Did you actually run the test?
2. `Akto uses a simple yet powerful DSL (Domain Specific Language)` - Akto doesn't use DSL 
3. The test case example you have provided is wrong and not sure where it's taken from. Did you login to akto and see how akto tests are written and run?
4. Setting up Akto instructions are wrong. Did you try to setup akto by yourself?

Please read the requirements above. I will be able to accept PR once the blog is factually correct. Let me know if you want help. Please make sure you have actually used the product before you write the blog. Incorrect blogs won't be accepted.

I am happy to help.

**Comment by Ankita28g** [comment]
@falconcode16 thanks for submitting this. Can you send me a link to your social where you have shared this blog?

**Comment by falconcode16** [comment]
@Ankita28g yes here it is - [Medium Link](https://medium.com/@rohitpatil1625/how-i-tested-for-crlf-injection-with-akto-2bd24bda047f)

**Comment by akash47angadi** [comment]
@Ankita28g, can you please assign it to me

**Comment by hridaya423** [comment]
Can i make a blog on this as well?

**Comment by avneesh-akto** [comment]
@falconcode16 Good job on the blog. Loved the creative analogy to explain CRLF in "The Hitchhiker’s Guide to CRLF Injection" part.

**Comment by Vishalk91-4** [comment]
@Ankita28g, can i make a blog also on it

**Comment by avneesh-akto** [comment]
Hi @Vishalk91-4.  @falconcode16 has already submitted a blog for it and it looks pretty good. Can you please pick some other issue

**Comment by RaagaAkto** [comment]
Hi @falconcode16 Please fill out[ this form here ](https://forms.gle/8d7B42XEJKePDwnn7)so we can send you Akto swags. Will let you know ETA of swags soon, thanks for your contribution! 🚀

**Comment by RaagaAkto** [comment]
Hi @falconcode16 , we've received your details, swags should reach you in a month!

**Comment by falconcode16** [comment]
Sure! Thank you.😊

**Comment by falconcode16** [comment]
> Hi @falconcode16 Please fill out[ this form here ](https://forms.gle/8d7B42XEJKePDwnn7)so we can send you Akto swags. Will let you know ETA of swags soon, thanks for your contribution! 🚀

A heartfelt thank you for the swags, they are just awesome!! I'm looking forward to continue my contribution to Akto✨✨
